lib/gitx/extensions/string.rb

```diff
@@ -4,9 +4,9 @@ def undent
     indent = scan(/^[ \t]*(?=\S)/).min.size || 0
     gsub(/^[ \t]{#{indent}}/, '')
   end
-  alias :dedent :undent
+  alias_method :dedent, :undent
 
   def blank?
-    self.to_s == ''
+    to_s == ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.